perfkitbenchmarker/providers/aws/aws_emr.py

```diff
@@ -61,19 +62,35 @@ def __init__(self, spark_service_spec):
       self.machine_type = DEFAULT_MACHINE_TYPE
     self.cmd_prefix = util.AWS_PREFIX + ['emr']
     if FLAGS.zones:
-      region = util.GetRegionFromZone(FLAGS.zones[0])
-      self.cmd_prefix += ['--region', region]
+      self.region = util.GetRegionFromZone(FLAGS.zones[0])
+    else:
+      cmd = ['aws', 'configure', 'get', 'region']
```

Comment:
I use the region as part of the logs bucket name when the logs bucket is created, which is why I wanted to get the explicit region string.